cidre/src/commonMain/kotlin/at/asitplus/cidre/IpAddressAndPrefix.kt

```diff
@@ -44,6 +45,39 @@ sealed interface IpAddressAndPrefix<N : Number, S : CidrNumber<S>> {
     /**`true` if this is (part of)  the [IpNetwork.SpecialRanges.multicast] network */
     val isMulticast: Boolean
 
+    /**
+     * Encodes this network into X.509 iPAddressName ByteArray (RFC 5280).
+     * IPv4: [4 bytes base address][4 bytes subnet mask]  (8 bytes)
+     * IPv6: [16 bytes base address][16 bytes subnet mask] (32 bytes)
+     */
+    fun toX509Octets(): ByteArray = address.octets + netmask
+
+    companion object {
+
+        /**
+         * Low-level helper used by [IpNetwork.fromX509Octets] and [IpInterface.fromX509Octets]
+         * to extract base address and CIDR prefix
+         * IPv4: [4 bytes base address][4 bytes subnet mask]  (8 bytes)
+         * IPv6: [16 bytes base address][16 bytes subnet mask] (32 bytes)
```

Comment:
This does not render correctly, because […] is just MD syntax for a href with an internal link target of the same name as the visible text